### PR TITLE
detective's gun is now two shots to down instead of fucking useless. why do you display PR names in OOC? i could just write the entire bee movie here and there's nothing god himself could do to stop me.

### PR DESCRIPTION
**Pull request by ghost**
compromise, since one shot to stun for a reloadable tazer was too strong according to the babies on /tg/, but making the det's gun blow ass for anything but murder (where it was then a stronger laser gun) wasn't particularly fair either

now it does 60 stamina damage, so it takes two shots to down someone, but doesn't apply a stun on its own
#### Changelog

:cl:  
tweak: the detective's gun takes two shots to stun someone now
/:cl:

### CODE REVIEW

**Comment by Nickvr628** [comment]
I like it

**Comment by Ratvar** [comment]
one shot to make someone drop anything they hold, tho
sounds good

**Comment by ghost** [comment]
two shots, and in case you forgot, that's also what a fucking tazer does

**Comment by Amelia0010** [comment]
put fuggin //yogs behind it you inbred australian farmer

**Comment by GuppyLaxx** [comment]
Please atomize your PR or include the damage reduction

**Comment by ghost** [comment]
no

File: code/modules/projectiles/projectile/bullets/revolver.dm [hunk 1]
```diff
@@ -14,8 +14,9 @@
 
 /obj/item/projectile/bullet/c38
 	name = ".38 bullet"
-	damage = 25
+	damage = 15
 	//knockdown = 60 //yogs - commented out
+	stamina = 60
```
Comment (ThatLing) [inline code/modules/projectiles/projectile/bullets/revolver.dm]: Not modular

**Review by ThatLing — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/projectile/bullets/revolver.dm (hunk 1 above)
Comment (ghost) [reply code/modules/projectiles/projectile/bullets/revolver.dm]: what the fuck does this mean

**Comment by GuppyLaxx** [comment]
PR denied

File: code/modules/projectiles/projectile/bullets/revolver.dm (hunk 1 above)
Comment (Bawhoppen) [reply code/modules/projectiles/projectile/bullets/revolver.dm]: It means it needs to use the modular codebase format. You're editing pure /tg/ code which is kept intact and unedited for purposes of staying up to date. Yogs then has other code files that override the /tg/ code as needed. This is supposed to be one of those files.

Comment (Bawhoppen) [reply code/modules/projectiles/projectile/bullets/revolver.dm]: I guess you don't need to do that for one line changes, however. So ignore me, sorry.

**Review by ThatLing — APPROVED** [review]

**Pull request merged**